docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -94,61 +94,90 @@ expands on what differentiates IBM, including microsimulation, from other types
 Advantages of microsimulation
 +++++++++++++++++++++++++++++
 
-.. todo::
-
-  Fill out the following sections under 'Advantages of microsimulation' subheading. 
-
 Correlation between risk exposures
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-At the individual level, risk exposures are frequently correlated. Examples include high body mass index and high fasting plasma glucose, tobacco smoking and alcohol use, and childhood height and weight.
+At the individual level, risk exposures are frequently correlated. Examples include high body mass index and high fasting plasma glucose, tobacco smoking and alcohol use, 
+and childhood height and weight. Microsimulation methods allow us to assign a risk exposure value independently to simulants, such that each one follows the desired population-
+level univariate distribution (the distribution of which comes from the GBD, in our microsimulations). In the common case of a dichotomous exposure, each simulant 
+(in the same age/sex/location group) has the same probability of exposure, which is equal to the prevalence in that population.
 
-Usually, Vivarium assigns each risk exposure independently to simulants, such that each one follows the desired population-level univariate distribution, which frequently comes from GBD. 
-In the common case of a dichotomous exposure, this means that each simulant (in the same age/sex/location group) has the same probability of exposure, which is equal to the prevalence in that population.
+Our cardiovascular disease (CVD) microsimulation offers a useful example of how risk exposure correlation typically looks in microsimulations. There are several correlated 
+risk factors that affect different CVD outcomes, including: body mass index (BMI), systolic blood pressure (SBP), low-density lipoprotein cholesterol levels (LDL-C), and fasting 
+plasma glucose (FPG). Using NHANES survey data, we calculated correlation coefficients for these relationships, which could then be used to generate propensity values for all
+simulants. As such, the correlation coefificients remain constant as simulants age and their risk exposure values change. You can read more about our modeling strategy for 
+these risk factor correlations :ref:`on this page <2023_sbp_ldlc_fpg_bmi>`.
 
 Continuous values
 ~~~~~~~~~~~~~~~~~
 
-e.g., allows the modeling of joint distribution between blood pressure and blood glucose levels -- this kind of thing is hard with categorical data characteristic of multiplication models
+Similarly, microsimulation allows the modeling of joint distribution between continuous values, something which is not as possible with categorical data characteristic of 
```

Comment:
This section feels like it is describing the same thing as the previous. I think it would be more distinct to point out that we can model a precise hemoglobin level for each simulant and leave it at that.